node-notes-cli/notes.js

```diff
@@ -0,0 +1,71 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+async function getJsonData() {
+  try {
+    const jsonData = JSON.parse(await readFile('./data.json'));
```

Comment:
This line of code is repeated several times. It would be nice to put it in a separate function.